src/Screens/App/Screens/BooksDashboard/Screens/BookList/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import Book from './Screens/Book';
+import books from './Resources/books';
+import './style.css';
+
+class BookList extends Component {
+
+  render() {
+    let check = (book) => {
```

Comment:
Be aware of creating new references in `renders` !
This should be an instance method (i.e. this should be placed outside this render)